apps/dashboard/app/controllers/projects_controller.rb

```diff
@@ -132,6 +137,12 @@ def templates
     end
   end
 
+  # Add icon URI as it is not provided by the user
+  def add_icon_uri
+    icon = project_params[:icon]
+    params[:project][:icon] = icon =~ /(fa[bsrl]?):\/\/(.*)/ || icon.nil? || icon.empty? ? icon : "fas://#{icon}"
+  end
```

Comment:
Seems like if this is shared it should be in some sort of module in a concern directory.

Moreover, I don't think it should modify the params, but instead take a string argument and return a string.